.github/workflows/validate-pr.yml

```diff
@@ -0,0 +1,18 @@
+name: "Validate PR"
+
+on:
+  pull_request_target:
```

Comment:
Use [pull_request_target](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request_target) instead of [pull_request](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request), the difference is explained [here](https://github.com/amannn/action-semantic-pull-request#event-triggers).